web/core/src/commonMain/kotlin/androidx/compose/web/css/CSSKT.kt

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
Not a big deal but it seems to be that this file can have better (= more intuituve) name )

Reply:
That object in  Browser API named just `CSS`, and it is kotlin implementation of that object... So i have another name CSSKotlinImpl.kt

Reply:
Do you have better naming for that?

Reply:
I usually take the "biggest" entity in the file and name the file accordingly but, again, it's no a big deal )